### ISSUE TITLE
Sending and receiveing data containing None

### ISSUE THREAD

**Issue by mortenengen** [specklesystems/specklepy#167]
### Expected vs. Actual Behavior

I am sending data through our Speckle server as a `dict` stored as an attribute of a `Base` object. The `dict` contains a bunch of key value pairs, and sometimes, a value might be `None`.

When sending this `Base` containing `None` in previous `specklepy` versions, it would be stored on the server with the value set as the string `null`, which I interpret to be a json version of the Python `None`. This is ok, but when receiving the object back, the value of the received object is still the string `null` and not `None`. This would cause hickups in my system on the receiving side, since as a safety measure I need to verify that the hash of the data is unchanged before processing the data.

After upgrading to `specklepy` 2.6.0, sending a `Base` with a value equal to `None` triggers an exception. See details on how to reproduce the behaviour below.

### Reproduction Steps & System Config

Python 3.9.7 and 3.10.1, specklepy 2.6.0, speckle.xyz 2.3.13

Initialize client and transport:
```python
from specklepy.api.client import SpeckleClient
from specklepy.api.credentials import get_default_account
from specklepy.transports.server import ServerTransport
from specklepy.api import operations
from specklepy.objects.geometry import Base

# Set host and stream id
host = 'https://speckle.xyz'
stream_id = '##########'

# Authenticate client and initialize transport
account = get_default_account()

client = SpeckleClient(host=host)
client.authenticate_with_token(token=account.token)

server_transport = ServerTransport(stream_id=stream_id, client=client)
```
Create and send object:
```python
# Send object
object_to_send = Base()
data = {'some_data_that_could_be_none': None}
object_to_send.data = data
object_id = operations.send(object_to_send, transports=[server_transport])
client.commit.create(stream_id=stream_id, object_id=object_id)
```

**Comment by izzylys** [comment]
heya, thanks so much for reporting this!

I'm about to merge a fix for this in #168 and will release 2.6.1 for ya shortly! you should now be able to serialize and deserialize nulls as expected, but let me know if you run into any other issues!

ps client now has an `authenticate_with_account` method you can use instead if you wish!

**Comment by mortenengen** [comment]
Great! Thanks for a speedy solution 😄 and thanks for the tip.
